components/user_settings/modal/user_settings_modal.tsx

```diff
@@ -326,10 +339,17 @@ class UserSettingsModal extends React.PureComponent<Props, State> {
                         componentClass='h1'
                         id='accountSettingsModalLabel'
                     >
-                        <FormattedMessage
-                            id='user.settings.modal.title'
-                            defaultMessage='Account Settings'
-                        />
+                        {this.props.globalHeaderEnabled && this.props.isContentChannelPreferences ? (
+                            <FormattedMessage
+                                id='channel_header.channelPreferences'
+                                defaultMessage='Channel Preferences'
+                            />
+                        ) : (
+                            <FormattedMessage
+                                id='user.settings.modal.title'
+                                defaultMessage='Account Settings'
```

Comment:
nit: maybe we should keep the naming consistent, account or profile.

Reply:
`Account Settings` is the Title of the modal. `Profile` is the tab. So was in the figma files.